dump-pack-ftp-unpack-restore/roles/dump-pack-ftp-data-sets/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - dump-pack-ftp-data-sets
```

Comment:
This file is missing a new line at the end, I would add that just for consistency.